packages/fuels-test-helpers/src/accounts.rs

```diff
@@ -85,6 +86,44 @@ pub async fn launch_custom_provider_and_get_wallets(
     Ok(wallets)
 }
 
+pub async fn connect_to_testnet_node_and_get_wallets(
```

Comment:
I wouldn't put these in the test helpers crate. The test helpers contain stuff we deem to be useful to all users. This is more of our internal desire to test the testnet. Why limit our users to stuff like:
* no more than 3 wallets
* the test net must be the official one from fuel (URL hardcoded)
* the env variables must be strictly the ones we set here (TEST_WALLET_SECRET_KEY_*) 
* etc

I'd rather we move this helper to our e2e tests and have it be private there since they are the only reason this exists.